lib/deaf_grandma.rb

```diff
@@ -19,9 +16,17 @@ def run!
 
 
   def speak(input)
+    "SPEAK UP SONNY!"
+  end
 
-    #Implement your code here <<<<<<<<<
-
+  def YELL(input)
+    @bye_counter += 1 if input.casecmp('bye').zero?
+    if @bye_counter <= 2
+      "NOT SINCE 1964!"
+    else
+      @bye_counter = 0
+      "SEE YOU LATER SONNY!"
+    end  
```

Comment:
Try to make it so that whether the user's speech counts as speaking or yelling depends on the nature of the input, not the method you're calling.